microsetta_interface/templates/account_details.jinja2

```diff
@@ -442,4 +453,40 @@
         {% endif %}
     </div>
 </div>
+{% if not admin_mode %}
+    {% if requested_deletion %}
+        <div class="container">
+            <div class="row">
+                <div class="col-12 {% if samples|length > 0 %}tooltipper{% endif %}" data-title="{{ _('Request account deletion.') }}">
+                    <p>
+                    </p>
+                        <center>
+                        _("Your account removal request is being reviewed. You will be notified via email once your account has been deleted.")
+                        </center>
+                </div>
+            </div>
+        </div>
+    {% else %}
+        <div class="container">
```

Comment:
A couple adjustments here:
* The request account deletion button is showing up on the account creation screen. It should only show once a user has finished setting up their account.
* Please move this section inside the card that contains their account details with a couple of <br /> tags between the "Save" button and the account deletion info.